tests/src/test/scala/system/basic/WskBasicTests.scala

```diff
@@ -88,7 +88,10 @@ class WskBasicTests
 
     it should "reject unauthenticated access" in {
         implicit val wskprops = WskProps("xxx") // shadow properties
-        val errormsg = "The supplied authentication is invalid"
+        //First it uses auth key verification, if failed, uses client certificate verification,
+        //for client certificate verification, if header doesn't include x-ssl-subject,it will
+        //throw CredentialsMissing Exception
+        val errormsg = "The resource requires authentication, which was not supplied with the request"
```

Comment:
If they're spray messages I've generally left those out of the singleton object for error messages which are generally the ones our system produces - in case we want to internationalize these in the future. It's ok to leave this Spray message out as a such.